### DISCUSSION TITLE
Kiwixarr

### DISCUSSION THREAD

**Discussion by ericnitschke** [kiwix/.github#16] [Ideas]
Until now I've been finding ZIM torrents [here ](https://library.kiwix.org/) and downloading via qbittorrent or transmission. After downloading I move the files to a different ZIM directory which stops the ZIM from seeding. But I would like to continue seeding, and would like to find a way for other users to do the same. 

The Arr suite uses [hardlinks](https://trash-guides.info/File-and-Folder-Structure/Hardlinks-and-Instant-Moves/#hardlinks-limitations) to keep the files in the downloads directory for seeding but also 'move' them to a relevant directory for usage without actually copying the file.  Is there anything similar to this for Kiwix? 

I'm imagining something like Kiwixseerr (a la [overseerr](https://overseerr.dev/) or [jellyseerr ](https://github.com/Fallenbagel/jellyseerr )) to browse ZIMs, then make requests to Kiwixarr (like sonarr/radarr/etc.) for organization and download via torrent client of choice.  Once downloaded files would be hardlinked to a directory watched by Kiwix Serve. In theory this would also enable auto-downloading/seeding of specific ZIMs or collections.

Pardon my elementary understanding of all of the above.  @kelson42 @Popolechien @rgaudin

**Comment by kelson42** [comment]
Why you don't open the ZIM file in Kiwix and leave it in your BitTorrent client directory? I believe I don't understand the problem you try to solve.

**Reply by ericnitschke** [reply]
A reasonable solution that underestimates the absolute chaos of my downloads directory. I guess I'd rather move ZIM files to a separate space for further curation like jellyfin libraries. ZIMs in various libraries/directories of interest while still ensuring seeding. Would like to know how others manage this.
